### ISSUE TITLE
bad longitudes in demo page

### ISSUE THREAD

**Issue by wysisoft** [mapbox/mapbox-gl-js#8257]
https://docs.mapbox.com/mapbox-gl-js/example/mouse-position/

gives bad longitudes as you cross over the dateline

https://jsfiddle.net/u8cowx7g/ is my ugly fix, maybe you guys have a better one?

**Comment by andrewharvey** [comment]
https://docs.mapbox.com/mapbox-gl-js/example/popup-on-click/ does something like your fix.

I think you could also use LngLat.wrap() https://docs.mapbox.com/mapbox-gl-js/api/#lnglat#wrap

**Comment by wysisoft** [comment]
wrap is the right answer I think, it should be added to the demo, perhaps both wrapped and unwrapped (if unwrapped still has purpose, for example in project/unproject)

**Comment by andrewharvey** [comment]
yep, let's leave this ticket open to track the improvement to the example

**Comment by colleenmcginnis** [comment]
I'm transferring this issue to the new home for Mapbox GL JS documentation ([/mapbox-gl-js-docs](https://github.com/mapbox/mapbox-gl-js-docs)). See https://github.com/mapbox/mapbox-gl-js/pull/8582 for details on this move.
